### PR TITLE
Handling word count <= 0

### PR DESCRIPTION
**Pull request by martinraison**
@yingjieMiao

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala [hunk 1]
```diff
@@ -61,10 +62,14 @@ class WordCountCommanderImpl @Inject()(
...
+  def getReadTimeMinutes(id: Id[NormalizedURI], url: String): Future[Option[Int]] = {
     getWordCount(id, url) map { estimate =>
-      val minutes = estimate / WORDS_PER_MINUTE
-      (READ_TIMES map { t => (t, Math.abs(t - minutes)) } min)._1
+      if (estimate <= 0) {
+        None
+      } else {
+        val minutes = estimate / WORDS_PER_MINUTE
+        Some((READ_TIMES map { t => (t, Math.abs(t - minutes)) } min)._1)
+      }
```
Comment (yingjieMiao) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala:72]: this is interesting. some alternative:
`READ_TIMES.dropWhile(_ < minutes).headOption.getOrElse(60)`

also, you might want to calculate Float minute first, then take ceil()

Comment (yingjieMiao) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala:72]: BTW, I don't think the code return what you want. this always returns 1?

Comment (eishay) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala:72]: When word count is very high (more then 15,000 words), are we always returning 60?

Comment (martinraison) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala:72]: Nevermind, I wrote this one a bit too quickly. I'm fixing that when I get
home. @eishay we return 60 but the front end will show "> 1h"

Comment (yingjieMiao) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala:72]: this is more like a UI issue. We can show something like 30 min+ or 60 
min+.

**Pull request merged**